submissions/fluidic-power/fluidic-power.md

```diff
@@ -67,12 +67,12 @@ caption: Table showing timing results of unmodified baseline benchmark compared
 
 </center>
 
-This shows that the effect that _all_ the extra fluid computations added by Fluidic Power in this benchmark is roughly `~0.15ms`! This number is probably still not very accurate because it's within the noise margins of the spaghetti base, the test is not ideal, and measuring time in parallel threads is innacurate. However, it does give us a good indication of where most processing time is going. This proves that the Electric Networks does have the biggest impact in on performance by far, and that the extra fluid calculations are almost neglible! It makes one wonder what performance we could achieve with the Electric Network not unnecesarily collecting statistics - perhaps enormeous Fluidic Power megabases with gigantic transformer substations next to your nuclear reactors.
+This shows that the effect that _all_ the extra fluid computations added by Fluidic Power in this benchmark is only roughly `~0.15ms`! This number is probably still not very accurate because it's within the noise margins of the spaghetti base, the test is not ideal, and measuring time in parallel threads is innacurate. However, it does give us a good indication of where most processing time is going. This proves that the electric network does have the biggest impact on performance by far, and that the extra fluid calculations are almost neglible. It makes one wonder what performance could be achieved if the electric network was not unnecesarily collecting statistics - perhaps enormous Fluidic Power megabases with gigantic transformer substations next to your nuclear reactors.
 
-All-in-all Fluidic Power performs much better than I expected. And any slowdowns perceived will mainly be caused by the many Electric Networks collecting statistics, and not the fluid computations itself. You will still likely be able to launch a rocket without having your UPS dropping below 60 UPS. You could probably even build quite a bit bigger if you build more efficiently (I've achieved 90SPM at 220UPS!), and the mod won't neccesarily TANK your UPS as Rseding predicted. Factorio is luckily really well designed and optimized, even when not running as it was intended. This proves that Fluidic Power is definitely playable performance wise, but that's notthe only aspect that counts.
+All-in-all, Fluidic Power performs much better than I expected. Any perceived slowdowns will mainly be caused by the many electric networks collecting statistics, and not the fluid computations itself. You will still likely be able to launch a rocket without having your UPS dropping below 60 UPS. You could probably even build quite a bit bigger if you build more efficiently (I've achieved 90SPM at 220UPS!), and the mod won't neccesarily *tank* your UPS as *Rseding1* predicted. Factorio is luckily really well designed and optimized, even when not running as intended. This proves that Fluidic Power is definitely playable performance-wise, but that's not the only aspect that counts.
```

Comment:
```suggestion
All in all, Fluidic Power performs much better than I had expected. Any perceived slowdowns will mainly be caused by the many electric networks collecting statistics, and not the fluid computations themselves. You will still likely be able to launch a rocket without having your UPS drop below 60 UPS. You could probably even build quite a bit bigger if you have an efficient layout (I've achieved 90SPM at 220UPS!), and the mod won't neccesarily *tank* your UPS as *Rseding1* predicted. Factorio is luckily really well designed and optimized, even when not running as intended. This proves that Fluidic Power is definitely playable performance-wise, but that's not the only aspect that counts.
```
Just wanted to explicitly mention that "All-in-all" is always "All in all" as I fixed it in part 1 as well, or at least the first page of Google results for `All-in-all` only shows `All in all` results. :-)